### ISSUE TITLE
`generator-star-spacing` should not be an error

### ISSUE THREAD

**Issue by stephenmathieson** [segmentio/eslint-config#51]
this super tedious and annoying.. theres no reason for it to fail builds

instead, i think we ought to require [20 spaces](https://github.com/segmentio/app/pull/1006/commits/9343f30b9681aa46c765a0072b386fdbd6c244b5) (note that that commit is green)

**Comment by ndhoule** [comment]
Yeah I was looking into this the other day and it seems like eslint doesn't let you require a space only when it's a named generator function, which seems inconsistent with how all their other spacing rules work.

This is also a fixable error by passing `--fix` to eslint, if your project doesn't already have a `fmt` make target you should add it, it's useful.

Feel free to PR this (and other quick fixes like this)
